account_invoice_export/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Account Invoice Export",
-    "version": "13.0.1.0.0",
+    "version": "13.0.1.1.0",
```

Comment:
Don't change the number, it will be changed automatically by the bot.

Reply:
Oh, I did not know that, thanks. Will fix